opentech/apply/funds/templates/funds/applicationsubmission_simplified_detail.html

```diff
@@ -0,0 +1,58 @@
+{% extends "base-apply.html" %}
+
+{% load wagtailcore_tags statusbar_tags %}
+
+{% block title %}{{ object.title }}{% endblock %}
+
+{% block body_class %}{% endblock %}
+
+{% block content %}
+{% if object.round.specific.is_sealed %}
+<ul class="messages">
+    {% include "includes/message_item.html" with message="This submission is sealed " tag="warning" close=False %}
+</ul>
+{% endif %}
+<div class="admin-bar">
+    <div class="admin-bar__inner">
+        <h1 class="beta heading heading--no-margin heading--bold">{{ object.title }}</h1>
+        <h5 class="heading heading--meta">
+            <span>{{ object.stage }}</span>
+            <span>{% if public_page %}<a class="link--transparent link--underlined" href="{% pageurl public_page %}" >{{ object.page }}</a>{% else %}{{ object.page }}{% endif %}</span>
+            {% if request.user.is_apply_staff and object.round %}
+                <span><a class="link--transparent link--underlined" href="{% url 'apply:rounds:detail' pk=object.round.pk %}">{{ object.round }}</a></span>
+            {% else %}
+                <span>{{ object.round }}</span>
+            {% endif %}
+            {% if request.user.is_apply_staff %}
+                <span>Lead: <a class="link--transparent link--underlined" href="{% url 'apply:submissions:list' %}?lead={{ object.lead.id }}">{{ object.lead }}</a></span>
+            {% else %}
+                <span>Lead: {{ object.lead }}</span>
+            {% endif %}
+        </h5>
+        {% status_bar object.workflow object.phase request.user author=object.user same_stage=True %}
```

Comment:
I dont think the status bar is relevant anymore - all submissions are accepted at this point - remove?

Reply:
Fixed and rebased.